src/main/java/de/teamlapen/vampirism/items/VampirismItem.java

```diff
@@ -39,4 +41,8 @@ protected void setTranslation_key(String name) {
         this.translation_key = Util.makeTranslationKey("item", new ResourceLocation(REFERENCE.MODID, name));
     }
 
+    public ITextComponent getCustomName() {
```

Comment:
Does this override something or is used for something else?

Reply:
From where are those methods called?

Reply:
nvm

Reply:
used in BloodPotionItem & HunterIntelItem for HunterTrainerScreen & Altar of Infusion for better looking code

Reply:
thx